tests/conftest.py

```diff
@@ -1664,6 +1665,13 @@ def finalizer():
                         namespace=ocsci_config.ENV_DATA["cluster_namespace"]
                     )
                     log.info("Ceph health check passed at teardown!")
+                    if ocsci_config.DEPLOYMENT.get("multi_storagecluster"):
+                        ceph_health_check_multi_storagecluster_external()
+                        log.info(
+                            "Ceph health check for multi-storagecluster external cluster passed at teardown!"
+                        )
+                        multi_storagecluster_external_health_passed = True
```

Comment:
I think that the variable should be initialized on the line above the `try` block line 1651. Because if the exception will be hit on the line 1669, then on the line 1690 you will fail on:
```
NameError: name 'multi_storagecluster_external_health_passed' is not defined
```